core/src/main/scala/ai/h2o/sparkling/backend/utils/H2OContextExtensions.scala

```diff
@@ -115,21 +115,34 @@ trait H2OContextExtensions extends RestCommunication with RestApiUtils with Shel
   }
 
   protected def getAndVerifyWorkerNodes(conf: H2OConf): Array[NodeDesc] = {
+    val h2oCluster = conf.h2oCluster.get + conf.contextPath.getOrElse("")
+    val h2oClusterName = conf.cloudName.get
     try {
-      lockCloud(conf)
+      var numberOfTries = 6
+      while (numberOfTries > 1) {
```

Comment:
this is confusing, this will perform the re-try only 5 times, while `numberOfTries` is 6
why not do the normal `var numberOfTries = 0; while (numberOfTries < 6)` and `numberOfTries += 1`?

Reply:
ok, will change to the proposed logic. Thanks!